packages/fe/pages/dataset/_id.vue

```diff
@@ -330,6 +342,15 @@ export default {
         { label: 'Storage Providers', value: this.storageProviderCount },
         { label: 'Locations', value: this.locations }
       ]
+    },
+    backButton () {
+      const url = this.mostRecentSearch
+      return {
+        url,
```

Comment:
Let's put `url` inside the object directly since it's not a variable that's being re-used.